### PR TITLE
Desktop Version Update

### PR DESCRIPTION
**Pull request by Kambuchi**
_**Mobile version adapted to desktop**_

-Backgrounds images w/ invisible boxes(for better positioning added
-Two project boxes added
-CSS update to several elements(width & display)

### CODE REVIEW

**Review by Shaher-11 — COMMENTED** [review]
## _Status:_ Invalid:x:
## Hello @Kambuchi 
This Code Review Request will be marked as invalid in your Dashboard as the linter checks are working as requested.
Please make the necessary changes and submit another code review request once you are ready! :smiley:

File: index.html [hunk 1]
```diff
@@ -22,16 +22,44 @@
...
+                </button>
+            </div>
         </section>
         <section class="projects">
-            <h2>Projects</h2>
-            <div class="project-box">
-                <img src="img/project-1.png" alt="project-1" class="project-img">
+            <div class="project-title">
+                <h2>Projects</h2>
+            </div>
```
Comment (Shaher-11) [inline index.html:36]: Kindly style this section as shown in the original design
right1

File: index.html [hunk 2]
```diff
@@ -22,16 +22,44 @@
...
-            <button>
-                <span>Start collaboration</span>
-            </button>
+            <div class="hero-content">
+                <h1>Hector<br>Pereira</h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+                <button>
+                    <span>Start collaboration</span>
+                </button>
+            </div>
```
Comment (Shaher-11) [inline index.html:31]: Reduce the image size in this section so it only fits the main page 
### current image size
wrong1

## Original design
right2

File: index.html [hunk 3]
```diff
@@ -128,39 +161,44 @@ <h3>Skills</h3>
...
+            <ul class="networks">
+                <li><img src="./img/ic_stackoverflow.svg" alt=""></li>
+                <li><img src="./img/ic_github.svg" alt=""></li>
+                <li><img src="./img/ic_twitter.svg" alt=""></li>
+                <li><img src="./img/ic_linkedin.svg" alt=""></li>
+                <li><img src="./img/ic_medium.svg" alt=""></li>
+                <li><img src="./img/ic_angellist.svg" alt=""></li>
+            </ul>
         </footer>
```
Comment (Shaher-11) [inline index.html]: Please style this section as the original one shown in the screenshot below
right3

## Current style
wrong3

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @Kambuchi  

#### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:blush::blush:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Kindly check the comments below.
- [ ] You need to make sure that your desktop version looks like the one in the Figma design [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

File: index.html (hunk 3 above)
Comment (Davidosky007) [reply index.html]: done 👍

File: index.html (hunk 2 above)
Comment (Davidosky007) [reply index.html:31]: fixed 👍

Comment (Davidosky007) [reply index.html:31]: Please notice that using the vw unit in the width property is causing an overflow on the website. Kindly keep in mind that vw stands for "viewport width" and this means that this element is going to take all the available space on the screen including the document margin. Please use the % unit instead, in order to fix the design issue.

using vw unit is making the background images bigger causing it to over flow.
Screenshot (50)
Screenshot (51)

File: index.html (hunk 1 above)
Comment (Davidosky007) [reply index.html:36]: Please notice that using the vw unit in the width property is causing an overflow on the website. Kindly keep in mind that vw stands for "viewport width" and this means that this element is going to take all the available space on the screen including the document margin. Please use the % unit instead, in order to fix the design issue.

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Kambuchi @smunozmo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
* I dropped some comment below, please read them and make the necessary changes 👍 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi Kambuchi and smunozmo! :wave: :wave:

####  Your project is complete! Nice job done for implementing the changes requested by the previous reviewer(s), your project looks great! Congratulations 🎉. 

## Good points 👍
No Linters errors ✔️
Follow the readme template ✔️
Semantics and syntax  are well used✔️


#### You may merge, good luck on your next project! :rocket:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**

- [OPTIONAL] Please ensure to implement necessary changes requested by previous reviewers. Reduce the image size in this section so it only fits the main page 

### current image size
wrong1

## Original design
right2 


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**